modules/apps/frontend-data-set/frontend-data-set-api/src/main/java/com/liferay/frontend/data/set/filter/FDSFilter.java

```diff
@@ -23,6 +25,8 @@
 
 	public String getLabel();
 
+	public Map<String, Object> getPreloadedData();
```

Comment:
If we use a `default` method here:
```
public default Map<String, Object> getPreloadedData() {
     return null;
}
```
then we would not need to override it in the Base classes, only just in those that need to return anything. Also, we'd not force other future Base classes to implement it unless needed

I don't remember well how this affects semver, we shall take a look to how this is applied in other interfaces